<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogQuality</class>
 <widget class="QDialog" name="DialogQuality">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quality Rules</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTreeWidget" name="trw_quality_rules">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>Rules for Points</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rules for Polygons</string>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
      <item>
       <property name="text">
        <string>Plots should not overlap</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Boundary nodes should be covered by Boundary Points</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable</set>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>Rules for Lines</string>
      </property>
      <item>
       <property name="text">
        <string>Boundary should not have dangles</string>
       </property>
      </item>
     </item>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btn_select_all">
     <property name="text">
      <string>Select All</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btn_clear_selection">
     <property name="text">
      <string>Clear Selection</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogQuality</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogQuality</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
